core/lib/spree/testing_support/dummy_app/routes.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 DummyApp::Application.routes.draw do
+  mount SolidusAdmin::Engine, at: '/admin' if defined?(SolidusAdmin) && ENV['LIB_NAME'] == 'solidus_admin'
```

Comment:
I wanted to investigate passing that somehow as an [argument for the `DummyApp`](https://github.com/nebulab/solidus/blob/5f9c960982146b4489d70c781637c1bc31bf4d91/core/lib/spree/testing_support/dummy_app.rb#L35) class. That would be cleaner, as here we're having the callee knowing about the caller, but I don't know if that would be possible... :thinking:

Reply:
Maybe we could pass a block to `.setup` for doing engine-local stuff before the app is initialized, but that would still be a one-off case just for solidus_admin as the other engines won't require it.

I think I've found a way to adjust the routes from within the `spec_helper`.